src/app/core/admin/admin-entity-details/admin-entity-panel-component/admin-entity-panel-component.component.html

```diff
@@ -1,7 +1,42 @@
 <div class="padding-regular">
-  <p>[ {{ config.component }} ]</p>
+  @if (entityConstructor?.ENTITY_TYPE) {
+    <mat-form-field appearance="fill" class="margin-bottom-regular">
+      <mat-label i18n>Entity Type</mat-label>
+      <mat-select
+        [(ngModel)]="selectedEntityType"
+        (ngModelChange)="onEntityTypeChange($event)"
+      >
+        @for (relatedEntity of availableRelatedEntities; track relatedEntity) {
+          <mat-option [value]="relatedEntity.entityType">
+            <app-entity-field-label
```

Comment:
this component is displaying fields of a type, not the type. Should this really be used here?